sqllineage/runner.py

```diff
@@ -42,6 +42,8 @@ def __init__(
         verbose: bool = False,
         silent_mode: bool = False,
         draw_options: Optional[Dict[str, str]] = None,
+        default_schema: Optional[str] = None,
+        tsql_no_semicolon: Optional[bool] = False,
```

Comment:
I'd prefer explicitly ask user to set value on SQLLineageConfig, and not through LineageRunner interface.